src/python/visclaw/data.py

```diff
@@ -289,7 +291,7 @@ def getgauge(self, gaugeno, outdir=None, verbose=True):
                     print "continuing..."
                     return None
                     
-                gauge_data = amrclaw.GaugeData()
+                gauge_data = amrclaw.GaugeData(self.num_dim)
                 gauge_data.read(outdir)
 
```

Comment:
Can we maybe have `num_dim` be an argument to `getgauge` or is that really not helping matters?